symtabAPI/src/Symtab.C

```diff
@@ -373,9 +373,20 @@ SYMTAB_EXPORT Symtab::Symtab() :
 
 SYMTAB_EXPORT bool Symtab::isExec() const 
 {
+    assert( is_a_out == obj_private->isOnlyExecutable() );
```

Comment:
If this is a class invariant, we should enforce it in the constructor.

Reply:
I will remove.  Present from early validation testing of the old and new code and I forgot to remove this.